auslib/web/admin/swagger/api.yaml

```diff
@@ -1893,82 +1916,956 @@ paths:
+        description: Returns scheduled changes for required signoffs of products.
+      parameters:
+        - $ref: '#/parameters/allSCParam'
+      responses:
+        '200':
+          description: Returns JSON object of list of all scheduled changes for required signoffs of products plus total count
+          schema:
+            type: object
+            required:
+              - scheduled_changes
+            allOf:
+              - $ref: "#/definitions/CountModel"
+              - type: object
+                properties:
+                  scheduled_changes:
+                    description: array where each element is a scheduled change for a Product Required Signoff.
+                    type: array
+                    minItems: 0
+                    items:
+                      type: object
+                      allOf:
+                        - $ref: '#/definitions/SCChangeType'
+                        - $ref: '#/definitions/SCTime'
+                        - $ref: '#/definitions/SCIdModel'
+                        - $ref: '#/definitions/SCDataVersion'
+                        - $ref: '#/definitions/SCGetModel'
+                        - $ref: '#/definitions/DataVersionNullable'
+                        - $ref: '#/definitions/SignoffsRequiredNullable'
+                        - $ref: '#/definitions/ProductModel'
+                        - $ref: '#/definitions/RoleModel'
+                        - $ref: '#/definitions/ChannelModel'
+                      required:
+                        - change_type
+                        - data_version
+                        - when
+                        - product
+                        - channel
+                        - role
+                        - signoffs_required
+    post:
+      summary: create scheduled change for required signoffs of Product.
+      description: >
+        Create scheduled change for required signoffs of product(existing or new). [Docs](http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#id34)
+      tags:
+        - Scheduled Changes
+      operationId: auslib.web.admin.views.mapper.scheduled_changes_rs_product_post
+      consumes:
+        - application/json
+      produces:
+        - text/html
+        - application/json
+      externalDocs:
+        url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#scheduled-changes-object"
+        description: "create scheduled change for required signoffs of product."
+      parameters:
+        - name: sc_rs_product_body
+          in: body
+          description: request body for sc required signoffs of product
+          required: true
+          schema:
+            type: object
+            allOf:
+              - $ref: '#/definitions/CSRFModel'
+              - $ref: '#/definitions/SCChangeType'
+              - $ref: '#/definitions/SCTime'
+              - $ref: '#/definitions/DataVersionModel'
```

Comment:
I just realized that data_version as Null isn't supposed to be passed at all since for change_type == 'insert' code[here](https://github.com/mozilla/balrog/blob/bf996837d2f19e88b9f9135a20b7e0563e11d559/auslib/web/admin/views/required_signoffs.py#L129) the wtfForm ScheduledChangeNewProductRequiredSignoffForm ([link](https://github.com/mozilla/balrog/blob/bf996837d2f19e88b9f9135a20b7e0563e11d559/auslib/web/admin/views/forms.py#L392) does not use data_version field at all. It looks like another UI (similar to rules) issue where extra arguments are passed as Null when they aren't supposed to be passed at all for a particular change_type. Looks like I need to add a TODO comment in swagger spec to fix UI later since data_version should not be passed as 'null' during a POST operation for 'update' or 'delete' change_types (since insert does not use data_version field at all).